Dockerfile

```diff
@@ -19,20 +19,23 @@ RUN ldconfig /opt/OpenBLAS/lib/
 ##################### PREREQUISITES ########################
 
 ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update && apt-get install -y pkg-config m4 zlib1g libplplot-dev plplot-driver-cairo
-RUN apt-get remove plplot-driver-qt -y
-RUN opam install alcotest utop
+RUN apt-get update -y
+RUN apt-get install -y m4 wget unzip aspcud libshp-dev libplplot-dev gfortran pkg-config git
+RUN cd /home/opam/opam-repository && git pull --quiet origin master
+RUN opam update -q
 
 ####################   INSTALL OWL  #######################
 
-COPY . owl
-WORKDIR /home/opam/owl
+ENV OWLPATH /home/opam/owl
 ENV OWL_DISABLE_LAPACKE_LINKING_FLAG 1
 ENV OWL_COMPILE_CFLAGS "-I/opt/OpenBLAS/include -I/home/opam/OpenBLAS/lapack-netlib/LAPACKE/include/ -L/opt/OpenBLAS/lib"
```

Comment:
I don't see why you still want to use this custom compilation, setting `PKG_CONFIG_PATH=/opt/OpenBLAS/pkgconfig` or wherever the custom openblas.pc is, should make it work fine